### PR TITLE
Add some codes in Accessibility sample app to work around bug 1155463

### PR DESCRIPTION
**Pull request by Amy-Li03**
Fix bug [1155463](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1155463?src=WorkItemMention&src-action=artifact_link), add some codes on sample app to make Narrator say "Please enter first name now" and added dataGridView.Focus() on "Click" event of "Add new" button.

## Proposed changes
Update related codes in DataControls.Designer.cs and DataControls.cs files

## Customer Impact
After this update, Narrator can read the status message as row added after click on Add New button.

## Regression? 

No

## Risk

No

### After

image

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/4132)

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/winforms?pullRequest=4132) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/winforms?pullRequest=4132) before we can accept your contribution.<br/><br/>:x: Amy-Li03 [sign now](https://cla.dotnetfoundation.org/dotnet/winforms?pullRequest=4132)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/winforms?pullRequest=4132) it.</sub>

**Review by RussKie — CHANGES_REQUESTED** [review]
How/why did "InteropTests/NativeTests/out/build/x64-Debug (default)" get committed? Please add "InteropTests/NativeTests/out" to .gitignore, and revert these files

File: src/System.Windows.Forms/tests/AccessibilityTests/DataControls.cs [hunk 1]
```diff
@@ -42,5 +42,13 @@ private void DataControls_Load(object sender, EventArgs e)
             if (cbox != null)
                 cbox.DroppedDown = true;
         }
+        private void bindingNavigatorAddNewItem_Click(object sender, EventArgs e)
+        {
+            dataGridView2.Focus();
+            this.bindingNavigator1.AccessibilityObject.RaiseAutomationNotification(
+                System.Windows.Forms.Automation.AutomationNotificationKind.Other,
+              System.Windows.Forms.Automation.AutomationNotificationProcessing.CurrentThenMostRecent,
+              "Please enter first name now");
```
[the remark this answers is not in the document]
Comment (Amy-Li03) [reply src/System.Windows.Forms/tests/AccessibilityTests/DataControls.cs]: Done

File: src/System.Windows.Forms/tests/AccessibilityTests/DataControls.cs [hunk 2]
```diff
@@ -42,5 +42,13 @@ private void DataControls_Load(object sender, EventArgs e)
             if (cbox != null)
                 cbox.DroppedDown = true;
         }
+        private void bindingNavigatorAddNewItem_Click(object sender, EventArgs e)
```
[the remark this answers is not in the document]
Comment (Amy-Li03) [reply src/System.Windows.Forms/tests/AccessibilityTests/DataControls.cs:47]: Done

**Comment by RussKie** [comment]
All of these files must be removed from the PR:
image

**Review by RussKie — APPROVED** [review]

**Pull request merged**